docker/dev/app/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:10.15.3-alpine
+
+RUN apk add --no-cache git yarn \
+    && yarn global add @vue/cli
+
+WORKDIR "/var/www"
+
+EXPOSE 8080
+
+CMD sleep 1000000000
```

Comment:
please replace `sleep` with `yarn serve` command

Reply:
Done
I do work locally, because using docker seems to require some hardware changes, and that would take some time (
Replaced the CMD anyway )